google-cloud-bigquerystorage/src/test/proto/jsonTest.proto

```diff
@@ -21,18 +21,20 @@ message ComplexRoot {
   optional int64 test_time_str = 15;
   repeated bytes test_numeric_repeated = 16;
   optional bytes test_numeric_str = 17;
-  optional bytes test_numeric_int = 18;
```

Comment:
why not just add the extra fields you wanted to the end of the previous list, instead of shuffle the old values and having to update other tests?

Reply:
readability counts I wanted it to read nicely rather that out of order.